.github/workflows/tests.yml

```diff
@@ -70,10 +70,24 @@ jobs:
         echo "SYMFONY_ENV=test" >> $GITHUB_ENV
         echo "MAUTIC_ENV=test" >> $GITHUB_ENV
 
+    - name: Set .env file to test
+      env:
+        mysql_port: ${{ job.services.mysql.ports[3306] }}
+        mariadb_port: ${{ job.services.mariadb.ports[3306] }}
+      run: |
+        cp .env.dist .env
+        if [[ "${{ matrix.db-types }}" == "mariadb" ]]; then
+          sed -i -r "s/^(DB_PORT=).*/\1$mariadb_port/" .env  
+        else
+          sed -i -r "s/^(DB_PORT=).*/\1$mysql_port/" .env  
+        fi
+
```

Comment:
Can you please explain what this is for? The name somehow doesn't match the command. Is this needed now after the cache clear?

Reply:
I will write a separate comment to explain the whole case